tests/04_00_02-fedchange.js

```diff
@@ -305,36 +324,34 @@ describe('RSK Federation change', function() {
 
   // this is not an actual test, this just modifies the blockchain state ensuring that the active federation contains several UTXOs
   it('generates several UTXOs in the active federation', async () => {
-    try{
+    try {
+
       const EXPECTED_UTXOS = 15;
-      let bridgeStatus = await getBridgeState(rskClientNewFed);
-      let existingUtxos = bridgeStatus.activeFederationUtxos.length;
-      amountOfUtxosToMigrate = existingUtxos < EXPECTED_UTXOS ? EXPECTED_UTXOS : existingUtxos;
+      const bridgeStatus = await getBridgeState(rskClientNewFed);
+      const existingUtxosCount = bridgeStatus.activeFederationUtxos.length;
+      amountOfUtxosToMigrate = existingUtxosCount < EXPECTED_UTXOS ? EXPECTED_UTXOS : existingUtxosCount;
 
       // Ensure there are enough UTXOs so that the migration need to be done in more than one transaction
-      if (existingUtxos < EXPECTED_UTXOS) {
+      if (existingUtxosCount < EXPECTED_UTXOS) {
+
         const UTXOS_TO_PAY_FEES = 1
-        const UTXOS_TO_TRANSFER = EXPECTED_UTXOS - existingUtxos;
-        var utxosToGenerate = UTXOS_TO_TRANSFER + UTXOS_TO_PAY_FEES;
-        var utxoValue = bitcoin.btcToSatoshis(1);
-
-        let activeFederationAddress = await getActiveFederationAddress();
-        var addresses = await pegClient.generateNewAddress('for_migration');
-        expect(addresses.inRSK).to.be.true;
-
-        await btcClient.sendToAddress(addresses.btc, utxoValue * utxosToGenerate);
-        await btcClient.generate(1);
-        await test.assertBitcoinBalance(addresses.btc, utxoValue * utxosToGenerate, "Wrong balance in account to migrate");
-        await wait(1000);
-        await sequentialPromise(
-          UTXOS_TO_TRANSFER, 
-          () => test.assertLock(addresses, [{ address: activeFederationAddress, amount: utxoValue }])
-        );
+        const UTXOS_TO_TRANSFER = EXPECTED_UTXOS - existingUtxosCount;
+        const utxosToGenerate = UTXOS_TO_TRANSFER + UTXOS_TO_PAY_FEES;
+        const utxoValueInSatoshis = bitcoin.btcToSatoshis(1);
+        const peginSenderAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
+        const totalAmountToSendInSatoshis = (utxoValueInSatoshis * utxosToGenerate) + (utxosToGenerate * btcEthUnitConverter.btcToSatoshis(btcTxHelper.getFee()));
```

Comment:
```suggestion
        const totalFeesInSatoshis = utxosToGenerate * btcEthUnitConverter.btcToSatoshis(btcTxHelper.getFee());
        const totalAmountToSendInSatoshis = utxoValueInSatoshis * utxosToGenerate + totalFeesInSatoshis;
```
Just to simplify a bit

Reply:
Updated!